haiku-games/peggy/peggy-0.5.2.recipe

```diff
@@ -0,0 +1,43 @@
+SUMMARY="A board game inspired by Mastermind(tm)"
+DESCRIPTION="In this game you have to crack a color code in 9 attempts\
```

Comment:
Needs a space before the \\-escaped newline or you'll see "...attemptsby getting...".

Reply:
Done. Thanks humdinger! :-)